### PR TITLE
 Make clickDoneEditingLink() in `test/pageobjects/InstructorFeedbackEditPage.java` more robust. #5455 

### LINKED ISSUE #5455: Make clickDoneEditingLink() in `test/pageobjects/InstructorFeedbackEditPage.java` more robust.
its very fragile now

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5455

### CODE REVIEW

**Comment by tarun14110** [comment]
ready for review

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 1]
```diff
@@ -729,9 +729,7 @@ public WebElement getStatusMessage() {
     }
     
     public InstructorFeedbacksPage clickDoneEditingLink() {
-        WebElement doneEditingLink = browser.driver.findElement(By.id("addNewQuestionTable"))
-                                                   .findElements(By.tagName("a"))
-                                                   .get(12);
+        WebElement doneEditingLink = browser.driver.findElement(By.id("button_done_editing"));
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: I prefer this to be a field of `InstructorFeedbackEditPage` itself. Use the `@FindBy` annotation to make this a field.

**Comment by wkurniawan07** [comment]
Just one minor suggestion.

**Comment by tarun14110** [comment]
@wkurniawan07 is it ok ?

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 2]
```diff
@@ -91,6 +91,9 @@
     @FindBy(id = "button_submit_add")
     private WebElement addNewQuestionButton;
     
+    @FindBy(id = "button_done_editing")
+    private WebElement doneEditingLink;
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Although this was named "Link" earlier, this is actually a "Button", so it's better if you name it that way.

**Comment by wkurniawan07** [comment]
Another minor change and sync and it should be good to go :+1:

**Comment by tarun14110** [comment]
@wkurniawan07 done

**Comment by wkurniawan07** [comment]
Looks good :+1: and since there's no other PR in the queue, I'll merge this in immediately.

**Pull request merged**